src/index.js

```diff
@@ -19,7 +23,7 @@ const startServer = async () => {
   const server = new ApolloServer({ typeDefs, resolvers });
 
   // Apply the express middleware if there are any
-  server.applyMiddleware({ app });
+  server.applyMiddleware({ app, cors: true });
```

Comment:
can you be certain that this is the correct syntax for the CORS middleware for apollo-server-express? :thankyoukindly:

I think you can just use `server.use(cors(corsOptions))`. So the only thing you have to define is the corsOptions after importing cors from cors.

Reply:
Check this out to see how to set corsOptions:

https://github.com/expressjs/cors#cors

Reply:
disabled the cors from the server.ApplyMiddleWare() and applied it via app.Use(cores(CorsOptions)). Alls added in the cors options variable right above it. It is empty though. Will look further into options